src/app/core/components/admin/services/admin-courses.service.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Observable} from 'rxjs';
+import {ICourse} from '../interfaces';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AdminCoursesService {
+
+  constructor(private http: HttpClient) {
+  }
+
+  addCourse(course): Observable<ICourse> {
+    return this.http.post<ICourse>('http://localhost:3002/courses', course);
```

Comment:
Replace it to constants. Viktoria should know how to do this. Ask her for help

Reply:
done